README.md

```diff
@@ -1,30 +1,96 @@
+
+**Development Environment** 
+I leveraged RStudio as the development environment due to its ability to support R programming, SQL, python, tools that streamline the coding process such as real-time code evaluation, debugging capabilities, and seamless integration with version control systems such as git/git hub. To document and present the analysis, I employed an R Markdown file. I chose R Markdown for its ability to integrate code with narrative, allowing for the creation of comprehensive reports that combine analysis with clear explanations. This feature enhances the presentation of technical concepts and results, enhancing communication of complex insights to both technical and non-technical audiences.
+
+**Data Extraction** 
+I initiated the data extraction process by establishing a database connection, using the DBI package to provide a consistent and standardized interface for querying and retrieving data from the database management system. Due to the lack of detailed information about the database’s origin, I was unable to fully configure the R script to connect directly to the specific data source. As an alternative, I employed the RPostgres package to illustrate connecting to a relational database. This approach enhances the code by demonstrating how to establish connections with a standard interface, improving the flexibility and adaptability of the script for future use when more precise database details become available.
+
+**Data Querying**
+The applicant data frame was generated by using SQL to query and extract data for analysis. The purpose of the SQL query was to extract features from the funnel table. This includes key metrics such as: unit_id a recruiting metric that identifies the different recruitment units, providing context for comparing performance across units; fiscal_year and fiscal_month offer temporal context, allowing for the examination of recruitment trends and patterns over time; and applicants represents the number of individuals applying, which serves as the target variable for the linear regression model.
+
+    **SQL Code Errors and Improvements**
+    
+    *Grouping Error:* Initially, the SQL code encountered an error related to grouping. The query was not            correctly aggregating the data by relevant dimensions, which could lead to inaccurate or incomplete             results. This issue was addressed by ensuring that the SQL query correctly specified the columns for            grouping, which is essential for aggregating data appropriately and obtaining meaningful insights.
```

Comment:
True, but what specifically did you do to the code to include the right dimensions?  I was hoping to get a changed `.sql` as part of the submission showing what would need to be changed.